client/components/forms/SignupForm.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import{Field, reduxForm} from 'redux-form';
+// need to add redux form validation
+// add styled div compenet
+function SignupForm (props){
+    const { handleSubmit,closeModal,userSignup,setActiveModal} = props;
+
+    return(
+        <div >
+            <form className="SignupForm" onSubmit={handleSubmit(values =>{
+                userSignup(values);
+                setActiveModal('confirm');
+                closeModal();
+            })}>
+                <label htmlFor="name">  Name </label>
+                <Field className="formInput" name="name" component="input" type="text"/>
+                <br/>
+                <label htmlFor="phone"> Phone #</label>
+                <Field className="formInput" name="phone" component="input" type="text"/>
+                <br/>
+                <label htmlFor="householdName"> Household name</label>
```

Comment:
Is the idea here that you’d choose a household name when signing up? Or that you could enter a household name that already exists? Or something else?